.circleci/config.yml

```diff
@@ -89,6 +89,16 @@ commands:
           name: Copy npmrc sample file to final location
           command: cp .npmrc.SAMPLE .npmrc
 
+  install-ruby-3-2-0:
```

Comment:
I'm wondering if we could use the official ruby orb: https://circleci.com/developer/orbs/orb/circleci/ruby#commands-install... If not we could also move this to our own orb, for more flexibility... But in any case, this is not a blocker. Happy to merge this as is.

Reply:
Right! I think it would be better. Honestly I took what was in the existing `ios` job as it was. But it's much cleaner, for sure. Will do that and try another dry-run with it to test it before. Thanks for the suggestion!